dev.env

```diff
@@ -62,4 +62,44 @@ export OP_ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
 # In general, the command to run a deployment script will look like this:
 # forge script script/deploymentScript/DeploymentScript.s.sol --rpc-url $SEPOLIA_RPC_URL --chain-id 11155111 --private-key $WALLET_DEPLOYER_PK --etherscan-api-key $ETHERSCAN_API_KEY --verify --broadcast --legacy -vvv
 
+
+# ------------------------------------------------------------------------------
+# Workflow Deployment Parameters
+
+# USP Workflow Deployment Parameters @todo change name of Workflow
+# Issuance Token
+export TOKEN_NAME_STRING=Test USP Yield Optimized Stablecoin
```

Comment:
Are these verified to work? on some systems this is a bit funky with strings and spaces so this could turn into a "Test" instead of "Test USP Yield Optimized Stablecoin". So i'd probably wrap it in single quotes to be safe.

Reply:
At least any strings, especially if they have spaces or other special characters like `-` or so. The rest can stay like it is.

Reply:
Good catch! Fixed it